src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -233,14 +233,14 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
   const promoImage = promoImageRawBlock?.model?.locator;
 
   const showTopics = Boolean(showRelatedTopics && topics.length > 0);
-
+  const authors = bylineLinkedData?.map(data => data?.authorName).join(', ');
```

Comment:
Use destructuing + add whitespace below
```suggestion
  const authors = bylineLinkedData?.map(({ authorName }) => authorName).join(', ');

```

Reply:
can keep the space below, but destrucutirng throws typescript errors. Because we used .filter(Boolean) on the original byline extractor, we had to keep |null inside the type for bylinelinkeddata because typescript doesn't correctly infer that there will no longer be any null values in this. In order to use destructing here we will have to add an unnecessary typeguard in here that will complicate the code again! Think its better to leave it as is with optional chaining, otherwise it will get messy and i hate the idea of filtering out null values twice 😭